CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Unreleased
+
+Adds new `getPagesTree` method that returns the nested pages in the right order with pieces that are also pages. To be able to build a sitemap page from any project.
```

Comment:
Please change to `getPageTree` for consistency with other usages of the term.

Reply:
DOne